raisinbread/RB_files/RB_user_perm_rel.sql

```diff
@@ -7,16 +7,12 @@ INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,3);
 INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,4);
 INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,5);
 INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,6);
-INSERT INTO `user_perm_rel` (`userID`, `permID`) VALUES (1,7);
```

Comment:
How did these get deleted from RB? There doesn't seem to be any `DELETE` patch in this PR to have been sourced..

Reply:
@driusan yes theres a cleanup
SQL/Cleanup_patches/2021-01-31-deleting_permissions.sql

Reply:
Oh weird, I don't know how I missed that with ctrl-F